vtr_flow/tasks/regression_tests/vtr_reg_nightly/vtr_reg_coffe/config/config.txt

```diff
@@ -0,0 +1,29 @@
+#
+############################################
+# Configuration file for running experiments
+##############################################
+
+# Path to directory of circuits to use
+circuits_dir=benchmarks/verilog
+
+# Path to directory of architectures to use
+archs_dir=arch/COFFE_22nm
+
+# Add circuits to list to sweep
+circuit_list_add=sha.v
+circuit_list_add=stereovision3.v
+
```

Comment:
These are simple circuits that don't have multipliers or RAM.  I suggest adding at least one design with multipliers and RAMs to test this architecture -- LU8PEEng would be a good choice (has both).
If you have a design that tests your new DSP modes I'd add it as well.

Reply:
Oh I had run a few others (arm_core and bgm in addition to these two). But they took over 11 hours each. I saw the time limit for nightly was ~6 hours, so, I removed those two. The time taken is very long because the layout is pretty big.

I didn't realize the two leftover ones don't have any RAM or DSP at all. I see that spree and or1200 are small and have  DSP and RAMs. I'll use those instead.

Should I add LU8PEEng as well? That will have to added to weekly regression though.

For the new DSP modes, I have designs that use a few modes - like floating point multiply, floating point mac, integer mac. I'll add a simple one (a matrix multiplier unit using BRAMs and DSPs in mac fp16 mode).